src/RefractDataStructure.cc

```diff
@@ -489,10 +592,27 @@ namespace drafter {
             }
         }
 
+        template <typename T>
+        struct MakeSectionInfo_ {
```

Comment:
If this is used someplace, I think this should have a better name and please move it to be with other section info stuff.